hub-client-images.js

```diff
@@ -78,25 +130,31 @@ javascript: (function () {
       existing_container.style.display = "block";
     }
   } else {
-    var links = document.getElementsByTagName("a");
     var container = document.createElement("div");
     container.id = "linked_images";
-    container.className = "link_to_mage";
-    container.width = "500px";
-    for (var link_i = 0; link_i < links.length; link_i++) {
-      var link = links[link_i];
+    container.className = "gyr-card-container";
+    // Loop through all the rows in the table of Hub documents
+    var formTableDataRows = document.querySelectorAll("table.data-table > tbody > tr")
+    formTableDataRows.forEach((row, index) => {
+      var tds = row.querySelectorAll("td");
+      // Assume 0th column has docType and 1st has link to file
+      var docTypeTd = tds[0];
+      var fileTd = tds[1];
+      var docType = docTypeTd.innerText;
+      var link = fileTd.querySelector("a");
+
       var href = link.href;
       var link_txt = link.innerText;
-      if (
-        /\.pdf$/i.test(link_txt) ||
-        /\.jpg$/i.test(link_txt) ||
-        /\.jpeg$/i.test(link_txt)
+      console.log(docType + ' ' + link_txt)
+      if (true // don't try to filter URLs
+        //    /\.pdf$/i.test(link_txt) ||
+        //    /\.jpg$/i.test(link_txt) ||
+        //    /\.jpeg$/i.test(link_txt)
       ) {
```

Comment:
Do we need this block?

Reply:
True.  We don't need it anymore, but it would be the place to skip some rows entirely, if that becomes necessary.  Let's leave it for the moment.